src/api.ts

```diff
@@ -47,6 +47,11 @@ export interface DirectoryDeletedEvent{
     fullPath:string
 }
 
+export interface FileSystemErrorEvent{
+    type:'fileSystemError',
+    error: Object
```

Comment:
Every type in the API must have an explicit serializable schema. using Object is really not helpful: how about something with with a message and a stack trace, both of them of type string?